### ISSUE TITLE
`make build` Process Stuck at Installing dependencies from lock file

### ISSUE THREAD

**Issue by iharshgor** [OpenDevin/OpenDevin#935]
#### Describe the bug

The `make build` command gets indefinitely stuck at the "Installing dependencies from lock file" step during the OpenDevin installation process on my Ubuntu system.

#### Setup and configuration

**Current version**:

```bash
commit 8ab9c6fb86db7ac21969fd5fadd18bab97bdb172 (HEAD -> main, origin/main, origin/HEAD)
Author: Engel Nyst <enyst@users.noreply.github.com>
Date:   Tue Apr 9 04:21:30 2024 +0200

    Revert the use of Router, good ole completion works. (#910)
    
    * Revert the use of Router, good ole completion works.
    
    * Stopgap exception message
    
    * Get the updated dependencies.
```

**My config.toml and environment vars** (be sure to redact API keys):

```toml
LLM_API_KEY="<YOUR OPENAI API KEY>"
WORKSPACE_DIR="./workspace"
```

**Commands I ran to install and run OpenDevin**:

```bash
git clone https://github.com/OpenDevin/OpenDevin
cd OpenDevin
make build
```

**Steps to Reproduce**:

1. Clone the OpenDevin repository.
2. Navigate to the OpenDevin directory.
3. Run `make build`.

**Logs, error messages, and screenshots**:
No specific errors are shown in the terminal. The process just halts without proceeding after displaying "Installing dependencies from lock file".

here are the bash logs:

```bash
$ make build
Building project...
Checking dependencies...
Checking Python installation...
Python 3.11.9 is already installed.
Checking npm installation...
npm 9.6.7 is already installed.
Checking Docker installation...
Docker version 26.0.0, build 2ae903e is already installed.
Checking Poetry installation...
Poetry (version 1.8.0) is already installed.
Dependencies checked successfully.
Pulling Docker image...
Using default tag: latest
latest: Pulling from opendevin/sandbox
Digest: sha256:b051045dd07803ec72da0342fca84d287f77c46ebde0ee2d57f551b863c8894e
Status: Image is up to date for ghcr.io/opendevin/sandbox:latest
ghcr.io/opendevin/sandbox:latest
Docker image pulled successfully.
Installing Python dependencies...
Installing dependencies from lock file

Package operations: 210 installs, 1 update, 0 removals

  - Installing certifi (2024.2.2): Pending...
  - Installing h11 (0.14.0): Pending...
  - Installing idna (3.6): Pending...
  - Installing sniffio (1.3.1): Pending...
  - Installing typing-extensions (4.11.0): Pending...
```

#### Additional Context

- Operating System: Ubuntu 22.04.4 LTS
- Kernel Version: 6.5.0-27-generic
- Python Version: 3.11.9
- Docker Version: 26.0.0
- NodeJS Version: 18.17.1
- Poetry Version: 1.8.0

I would appreciate any guidance or suggestions on how to resolve this build issue. Thank you in advance for your help and support.

**Comment by iharshgor** [comment]
The installation issue was resolved by updating Poetry from 1.8.0 to 1.8.2. Thanks for the assistance. Closing this issue.

**Comment by enyst** [comment]
Thanks for sharing!
